tests/unit/commands/buildcmd/test_build_context.py

```diff
@@ -558,17 +558,19 @@ def test_must_exclude_functions_from_build(
         self.assertEqual(context.stacks, [stack])
         self.assertEqual(context.is_building_specific_resource, bool(resource_identifier))
         ctx_resources_to_build = context.resources_to_build
+        final_resources = [x for x in resources_to_build if x not in excluded_resources] if excluded_resources else []
```

Comment:
hmm I think it shouldn't be empty list if there's no excluded resources? Maybe you meant ... if excluded_resources else resources_to_build?

Reply:
also could we rename to `non_excluded_resources` just to be clear 😅  (or maybe it's just me)

Reply:
Yes that is my mistake in logic, it should be `resources_to_build`. Will clear that up in the next commit!